### ISSUE TITLE
Issue with updog not resolving images and blocking website

### ISSUE THREAD

**Issue by njtierney** [rbind/njtierney.com#29]
As pointed out here 

https://twitter.com/Arfness/status/1187615886745788416?s=19

This is quite a shame as all my talks are hosted through updog.

@yihui have you run into this issue?

**Comment by yihui** [comment]
Hmm. I have never run into this. Did Jess reply your email last time?

**Comment by njtierney** [comment]
No, when I emailed updog it bounced back saying that updog is linked with
spam, and they didn't respond on twitter so I wasn't really sure how to
even get in touch with them?
